packages/plugin-pack/sources/index.ts

```diff
@@ -39,6 +39,9 @@ const beforeWorkspacePacking = (workspace: Workspace, rawManifest: any) => {
     if (rawManifest.publishConfig.exports)
       rawManifest.exports = rawManifest.publishConfig.exports;
 
+    if (rawManifest.publishConfig.imports)
+      rawManifest.exports = rawManifest.publishConfig.imports;
```

Comment:
```suggestion
      rawManifest.imports = rawManifest.publishConfig.imports;
```

Reply:
A great example if why even super-small PRs should have tests 🙈